.travis.yml

```diff
@@ -20,14 +20,16 @@ env:
       - OS=fedora DIST=26
       - OS=fedora DIST=27
 
-install:
-  - sudo apt-get install coreutils
-
 script:
+  # lulpeg tags have the following format: major.minor
+  # They are not annotated and so can not be used by packpack
+  # automaticaly. Also packpack excpects major.minor.patch tag format.
+  # That is why we have to pass VERSION and RELEASE.
   - git describe --tags
-  - VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | sed 's#-#\.#g' | cut -c 2-`
   - git clone https://github.com/packpack/packpack.git packpack;
-  - VERSION=$VERSION packpack/packpack;
+  - VERSION=`git describe --tags | sed 's/\(.*\)-.*/\1/' | cut -c 2- | cut -d '-' -f-1`
+  - RELEASE=`git describe --tags | sed 's/\(.*\)-.*/\1/' | cut -c 2- | cut -d '-' -f2`
```

Comment:
Is it ok that VERSION is 0.1 (have not patch level)? If so, you need to change the description above (and also my proposed version), because it stated that major.minor.patch is mandatory format for a tag and VERSION.

Reply:
It's ok, packpack pack pages like lulpeg-0.1-7 in this case (7 is commits count)
Fixed description

Reply:
I confused things.
Here is the citation from github/packpack/packpack:
_Create an annotated major.minor git tag in your repository. PackPack will automatically set patch level based on the commit number from this tag in order to provide major.minor.patch semantic versioning:_
```
$ git tag -a 1.0
$ git describe --always --long
1.0-0-g5c26e8b # major.minor-patch = 1.0-0
$ git push origin 1.0:1.0 # Push to GitHub
```

Reply:
I think that versioning schema can varies between different repositories. When a repository uses major.minor tags, then we set patch level to commits count above the tag, but when a repository uses major.minor.patch, then patch level is extracted from the repository, but additional 4th number with commits count is attached.

Reply:
Discussed verbally. Ivan bring the fact that RELEASE means package release version, which is used to leave official version as is and increase package version. In our case it should be always 1. Or maybe always 2, because we slightly changed the package and its patch level version (commits count).